fedimint-client/src/envs.rs

```diff
@@ -0,0 +1,26 @@
+use std::str::FromStr;
+use std::time::Duration;
+
+use tracing::warn;
+
+pub const FM_DISCOVER_API_VERSION_TIMEOUT_ENV: &str = "FM_DISCOVER_API_VERSION_TIMEOUT";
+
+#[cfg(not(target_family = "wasm"))]
+pub fn get_discover_api_version_timeout() -> Duration {
```

Comment:
This commit was in a different PR already, I guess it will be dropped by GH on merge? If you do another round would be good to drop manually.